src/components/Clients/ClientList.jsx

```diff
@@ -0,0 +1,49 @@
+// React Imports
+import React, { useContext } from 'react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+// Context Imports
+import { UserListContext } from '../../contexts';
+// Component Imports
+import ClientListTable from './ClientListTable';
+
+/**
+ * ClientList Component - Component that generates ClientList section for PASS
+ * which interfaces with Solid Pod to fetch user list
+ *
+ * @memberof Clients
+ * @name ClientList
+ */
+
+const ClientList = ({ loadingActive }) => {
```

Comment:
The loadingActive state in Line 20 is no longer in use. Can be removed without affecting it.

Reply:
Done